docs/guides/integration-aws-entity-resolution.md

```diff
@@ -153,7 +154,7 @@ To create the matching workflow, first sign in to the AWS Management console, op
      |0001|Test 1|test1@uidapi.com|1/1/90|
      |0002|Test 2|test2@gmail.com|1/2/78|
    
-      NOTE: There can be only **one** email or phone number field per record. If email is present, phone number is treated as pass-through. To process phone numbers, create a separate workflow. See details earlier in this section.
+      NOTE: If you have both email and phone numbers in the same record, the workflow will duplicate each record in the output. If you don't prefer this, then the best approach is to create separate workflow for each. See details earlier in this section.
```

Comment:
NOTE: If you have both email and phone numbers in the same record, the workflow duplicates each record in the output. If you don't want this, the best approach is to create a separate workflow for each. See details earlier in this section.

(again, just using present tense plus fixing "don't prefer" wording)